<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SmartCalc v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    font-family: 'Times New Roman', Times, serif;
    font-size: 20px;
    margin: 10px;
    background-color:#484545
}

QFrame#frame_lines {
    padding: 10px;
    background-color:#505050;
}
QFrame#frame_buttons {
    background-color: #1C1C1C;
}
QLineEdit {
    padding: 10px;
	 color: white;
}

QLabel {
    padding: 10px;
    color: white;
}


QPushButton#btn_1, #btn_2, #btn_3, #btn_4, #btn_5, #btn_6, #btn_7, #btn_8, #btn_9, #btn_0, #btn_dot{
    background-color: #d4d4d2;
	 color: black;
	padding: 10px;
	border-radius:10px

}

QPushButton#btn_div, #btn_mul, #btn_plus, #btn_sub, #btn_equal{
    background-color: #FF9500;
	 color: black;
	padding: 10px;
	border-radius:10px
}

QPushButton#btn_acos, #btn_asin, #btn_atan, #btn_clear_line, #btn_cos, #btn_credit, #btn_deposit, #btn_left_bracket, #btn_ln, #btn_log, #btn_plot, #btn_pow, #btn_right_bracket, #btn_sin, #btn_sqrt, #btn_tan, #btn_remove_symbol, #btn_unary_plus_minus, #btn_mod{
    background-color: #505050;
	 color: white;
	padding: 10px;
	border-radius:10px
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="layout_lines">
      <item>
       <widget class="QFrame" name="frame_lines">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_result">
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>255</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_buttons">
      <item>
       <widget class="QFrame" name="frame_buttons">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QGridLayout" name="layout_functions">
           <item row="2" column="0">
            <widget class="QPushButton" name="btn_acos">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>acos(x)</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_left_bracket">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>(</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_cos">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>cos(x)</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_right_bracket">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>)</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="btn_sqrt">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>sqrt(x)</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btn_pow">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>^</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="btn_clear_line">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>C</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="btn_plot">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>Plot</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="btn_credit">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>Credit Mode</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="btn_deposit">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>Deposit Mode</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="btn_log">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>log(x)</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btn_ln">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>ln(x)</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btn_asin">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>asin(x)</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_sin">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>sin(x)</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btn_tan">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>tan(x)</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="btn_atan">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>atan(x)</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="layout_digits_and_opers">
           <item row="2" column="0">
            <widget class="QPushButton" name="btn_4">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: white
}</string>
             </property>
             <property name="text">
              <string>4</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_remove_symbol">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>AC</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_7">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: white
}</string>
             </property>
             <property name="text">
              <string>7</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="btn_1">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: white
}</string>
             </property>
             <property name="text">
              <string>1</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btn_mod">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>%</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_unary_plus_minus">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: gray
}</string>
             </property>
             <property name="text">
              <string>x</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="btn_div">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: orange
}</string>
             </property>
             <property name="text">
              <string>/</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="btn_mul">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: orange
}</string>
             </property>
             <property name="text">
              <string>*</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="btn_sub">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: orange
}</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="btn_plus">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: orange
}</string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QPushButton" name="btn_equal">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: orange
}</string>
             </property>
             <property name="text">
              <string>=</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="btn_dot">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: white
}</string>
             </property>
             <property name="text">
              <string>.</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="btn_3">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: white
}</string>
             </property>
             <property name="text">
              <string>3</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btn_2">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: white
}</string>
             </property>
             <property name="text">
              <string>2</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btn_5">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: white
}</string>
             </property>
             <property name="text">
              <string>5</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_8">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: white
}</string>
             </property>
             <property name="text">
              <string>8</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btn_9">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: white
}</string>
             </property>
             <property name="text">
              <string>9</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="btn_6">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: white
}</string>
             </property>
             <property name="text">
              <string>6</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="btn_0">
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
background-color: white
}</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
